harmonia/src/main.rs

```diff
@@ -142,6 +142,22 @@ fn extract_filename(path: &str) -> Option<String> {
     }
 }
 
+fn cache_control_max_age(max_age: u32) -> http::header::CacheControl {
+    http::header::CacheControl(vec![http::header::CacheDirective::MaxAge(max_age)])
+}
+
+fn cache_control_max_age_1y() -> http::header::CacheControl {
+    cache_control_max_age(365 * 24 * 60 * 60)
+}
```

Comment:
For immutable content, the best practice is to set `Cache-Control: max-age=365000000, immutable`. I don't know that it will make a big difference in practice but is semantically more accurate.

See https://bitsup.blogspot.com/2016/05/cache-control-immutable.html